src/nnue/layers/affine_transform_sparse_input.h

```diff
@@ -23,6 +23,7 @@
 
 #include <algorithm>
 #include <cstdint>
+#include <cstddef>
```

Comment:
(Nitpicking) You can keep the `#include`s sorted:
```suggestion
#include <cstddef>
#include <cstdint>
```